circle.yml

```diff
@@ -13,4 +13,4 @@ deployment:
       - git config --global user.name "Apiary Bot"
       - git config --global user.email "support@apiary.io"
       - git remote set-url origin "https://${GH_TOKEN}@github.com/apiaryio/apiaryio.github.com"
-      - rake deploy
+      - bundle exec rake deploy
```

Comment:
This means that users using the Rakefile do not need to worry about running it via `bundle exec`. Currently since the htmlproofer commands these instructions on README are incorrect because none of them use `bundle exec`: https://github.com/apiaryio/apiaryio.github.com#rake-usage.